cloudcix_primitives/storage_lxd.py

```diff
@@ -1,31 +1,75 @@
 """
-Primitive for Storage drives on LXD hosts
+Module for updating the root disk size of an LXD container.
 """
 # stdlib
 from typing import Tuple
-# lib
+# libs
+from cloudcix.rcc import API_SUCCESS, CHANNEL_SUCCESS, comms_lxd
 # local
-
+from cloudcix_primitives.utils import HostErrorFormatter, LXDCommsWrapper
 
 __all__ = [
-    'build',
-    'read',
-    'scrub',
     'update',
 ]
 
+def update(
+        endpoint_url: str,
+        project: str,
+        instance_name: str,
+        new_size: str,
```

Comment:
To be compatible with build and ensure all sizes are in the same measurement, we pass in size as an integer and append `GB` to the root device size when calling the LXD API client. -https://github.com/CloudCIX/primitives/blob/df2009d4e32d9560f11c2850d5c54a864e3c7623/cloudcix_primitives/lxd.py#L184

Can the same pattern be followed here?